docs/ABI/Mangling.rst

```diff
@@ -199,6 +200,7 @@ types where the metadata itself has unknown layout.)
   global ::= assoc-type-name 'Tl'        // associated type descriptor
   global ::= assoc-type-name 'TM'        // default associated type witness accessor (HISTORICAL)
   global ::= type assoc-type-list protocol 'Tn' // associated conformance descriptor
+  global ::= type type protocol 'Tn' // inherited protocol witness descriptor
```

Comment:
Same question about ambiguity.  Also, I think a better name for this would be "base conformance descriptor", as saying "protocol witness" invites confusion with the more general concept.

Reply:
I think it's unambiguous, but I agree that it's safer to simply burn another two mangling characters here. Will follow up once testing here completes.